metadata.rb

```diff
@@ -1,20 +1,23 @@
-name             'cerner_splunk'
-maintainer       'Healthe Intent Infrastructure - Cerner Innovation, Inc.'
-maintainer_email 'splunk@cerner.com'
-license          'Apache-2.0'
-description      'Installs/Configures Splunk Servers and Forwarders'
-long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.14.0'
+# frozen_string_literal: true
 
-source_url       'https://github.com/cerner/cerner_splunk'
-issues_url       'https://github.com/cerner/cerner_splunk/issues'
+name              'cerner_splunk'
+maintainer        'Healthe Intent Infrastructure - Cerner Innovation, Inc.'
+maintainer_email  'splunk@cerner.com'
+license           'Apache-2.0'
+description       'Installs/Configures Splunk Servers and Forwarders'
+long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+version           '3.0.0'
 
-chef_version     '~> 12.4' if respond_to?(:chef_version)
+source_url        'https://github.com/cerner/cerner_splunk'
+issues_url        'https://github.com/cerner/cerner_splunk/issues'
 
-depends          'chef-vault', '~> 1.3'
-depends          'ulimit', '~> 0.3'
-depends          'xml', '~> 1.2'
+depends           'chef-vault', '~> 3.0'
+depends           'cerner_splunk_ingredient'
```

Comment:
This should probably be pessimistic on major version.  '~> 1.0'

Reply:
The version is presently 0.1.0, though I suppose I should change that to 1.0 in ingredient too.

Reply:
ecc3ee815cae26cb5fc0ddb3b51cfb7ea76a20c9